### DISCUSSION TITLE
differences between fov-mono3d and mv-mono3d

### DISCUSSION THREAD

**Discussion by hanssssssss** [open-mmlab/mmdetection3d#2064] [Q&A] [answered]
Hi, I am confused about the difference between the fov-mono3d and mv-mono3d on the waymo dataset. I found that the number of data being trained is less than that in the dataset. Could you please me more information about that? Thanks.

**Comment by ZCMax** [comment, marked as answer]
fov-mono3d means we only use the front view camera images for training mono3d model, mv-mono3d means we use all the camera images for training mono3d model

**Reply by hanssssssss** [reply]
Thanks for the quick reply. I am also confused about the number of data being trained. For instance, the number of data per epoch under mv-mono3d setting on waymo dataset is 7905*batchsize(3), which is less than the number of data in any folder(image_0, image_1 ....) in the folder.
besides, in my understanding, image_0 folder should be the front view image, but the the number of data being trained can not match the number of data in the folder.
![image](https://user-images.githubusercontent.com/57829884/204434256-4c34b146-e5c0-4641-8f17-0209a4fd97dc.png)

**Reply by ZCMax** [reply]
the load_interval is set to be 5, meaning using one image every 5 images. Besides, the `batch_size` is not 3 during training, if you are using the config of https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/configs/pgd/pgd_r101_fpn-head_dcn_16xb3_waymoD5-mv-mono3d.py, the real batch_size should be 16x3=48.

**Reply by hanssssssss** [reply]
Thanks! Btw, I did not use the default config files.
